### ISSUE TITLE
Overlay displays no parameter

### ISSUE THREAD

**Issue by DevTechProfile** [GPUOpen-Tools/OCAT#48]
Version: 1.2.0.167, GeForce 1080 Ti with latest driver, Windows 10 with latest updates
No parameter are been displayed when overlay is active, in no game. It's completely "dead". Paint.net is the only exception. Not a quite good shooter... ^^

**Comment by aurolou** [comment]
Unfortunately, we couldn't reproduce the issue. Do you mind providing some more details, e.g. name some game titles and list the repro steps on how you enable the overlay. Are you already on RS5? Thanks

**Comment by DevTechProfile** [comment]
I'v cleaned up my system. I've uninstalled all versions (there was also an older version), deleted all registry entries. So for me nearly everything is fine. On my system with GeForce 1080 Ti and latest driver:

Raft (Unity) -> works
Earthfall (Unreal Engine) -> works
SotTR (Crystal Dynamics) -> works
Doom (id Tech) -> doesn't work
Wolfenstein II (id Tech) -> works
Vampyr (Unreal Engine) -> works
Call of Duty: WWII (IW) -> works
Strange Brigade Vulkan -> works

But, there are users with AMD Vega 64 graphic cards, who have still problems after clean up:

Vermintide 2 -> doesn't work
Frostpunk -> doesn't work
Forza -> doesn't work
Unity engine -> works
Frostbite engine -> works
Azzura engine -> works
Quake Champions -> works

**Comment by aurolou** [comment]
I was able to reproduce the issue on Frostpunk using the global hook - no overlay is shown because it seems the hooking happens too late, once everything is already set up. However, the exclusive hook (under the tab 'Launch App') should work.
I couldn't repro the issues on Vermintide 2 - could you provide please more details? Also for Forza, I unfortunately only have the UWP version, which is troublesome in its own way, so more details about the issue would be very helpful.

**Comment by Taxxor90** [comment]
I'm also a Vega owner, now on Version 1.2.0.185

Out of the games I've installed, these work with the exklusive hook, any chance getting it to auto hook up sooner?

Frostpunk 
Ni No Kuni 2

And these don't work at all
Final Fantasy XV
Starcraft 2, Stronghold Crusader 2, Borderlands 2, They are Billions (DX9?)
Destiny 2
Spellforce 3 (Overlay shows, but extreme graphic errors, nearly a white screen in the menu, without the overlay it works fine)

**Comment by aurolou** [comment]
Could you try launching Final Fantasy XV with the command line argument steam://run/637650 ? This should prevent the app to restart and make the exclusive hook work.

I will have a look at the other games you mentioned.
Unfortunately, I haven't found a solution yet to get the auto hook up sooner, still working on it.

Thanks for the testing!

**Comment by Taxxor90** [comment]
Like putting exatxcly "steam://run/637650" into the launch options in steam? This didn't work.

**Comment by aurolou** [comment]
Not in Steam but in the command line arguments field of OCAT, when you launch the executable via the 'Launch App' tab.

**Comment by Taxxor90** [comment]
Okay that worked, now the overlay is showing from the start.

While this is a solution if I really want to use OCAT for FFXV, I wouldn't choose it over FRAPS because with FRAPS I get the overlay without setting the specific game up first.

So as long as the auto hook up doesn't work with all the games, OCAT will probably be used only for DX12 and Vulkan Games, where the FRAPS overlay doesn't work.

Which is a shame becaus I really like the OCAT overlay and the visualize option.

**Comment by Taxxor90** [comment]
I get the capture file for NMS as well now, maybe it didn't work with the previous version  or I did something wrong.

**Comment by aurolou** [comment]
Yes, that is the normal behaviour. I do make a note though that it seems to cause some confusion.

**Comment by Taxxor90** [comment]
May I ask why this is the case? It seems quite inconvenient to set the folder everytime you start OCAT. My plan was to have it on auto start for overlay and logging.

**Comment by rys** [comment]
We'll add saving any retargeted folder location to a future release (and note the current behaviour in the documentation until then). Tracked as #53
